Runtime/MeshSimplifier.cs

```diff
@@ -1177,18 +1177,17 @@ private void UpdateReferences()
                 int v2 = triangles[i].v2;
                 int start0 = vertices[v0].tstart;
                 int count0 = vertices[v0].tcount;
+                ++vertices[v0].tcount;
```

Comment:
```suggestion
                int count0 = vertices[v0].tcount++;
```

Reply:
@is3d-1 Did you not agree with the suggestions? since you resolved them without applying or commenting

Reply:
Yes I do ! I'm just not familiar with Pull Request on Github. I think I did not click on the right buttons... Let me try another one...